stac_fastapi/types/stac_fastapi/types/config.py

```diff
@@ -22,6 +22,11 @@ class ApiSettings(BaseSettings):
     # `pydantic.BaseSettings` instead
     default_includes: Optional[Set[str]] = None
 
+    stac_fastapi_title: str = "stac-fastapi"
+    stac_fastapi_description: str = "stac-fastapi"
+    stac_fastapi_version: str = "0.1"
+    stac_fastapi_landing_id: str = "stac-fastapi"
```

Comment:
as discussed in #639 I think it makes more sense for this variable to be named `landing_id`